config/redirects.json

```diff
@@ -2298,5 +2298,10 @@
         "source": "/products/atlas/query-multiple-databases-with-atlas-data-lake/",
         "destination": "/products/atlas/query-multiple-databases-with-atlas-data-federation/",
         "permanent": true
+    },
+    {
+        "source": "/products/mongodb/generate-mql-with-mognosh-and-openai/",
+        "destination": "/products/mongodb/generate-mql-with-mongosh-and-openai/",
+        "permanent": true
```

Comment:
Just let me know when you promote this, I need to make this change in Strapi when you do.

Reply:
will do, probably sometime in the afternoon